### PR TITLE
Feature/1 bbc clean data extractor

### PR DESCRIPTION
**Pull request by frenchfaso**

### CODE REVIEW

File: bbc_clean/treerez/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# TreeRez
```
Comment (zgypa) [inline bbc_clean/treerez/README.md:1]: Maybe you could add the reason why we needed to develop this tool in the first place.

File: bbc_clean/treerez/errors.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+ERROR:root:ERROR opening: F:/Bolton Backup/Bolton-Brush Study Files/Backup Broadbent Bolton Collection of Patient X-rays\B3582\Frontal\B3582F10.tif
```
Comment (zgypa) [inline bbc_clean/treerez/errors.txt:1]: Remove if we no longer need this file.

File: bbc_clean/treerez/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+numpy==1.24.2
```
Comment (zgypa) [inline bbc_clean/treerez/requirements.txt:1]: Any reason to lock to specific versions for this simple tool?

File: bbc_clean/treerez/treerez.py [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+def get_scans(patients_sample) -> list:
+	""" Return a list with all the scans of the sampled patients """
+	
+	scans = []
+	for patient in patients_sample:
+		xray_types = os.scandir(os.path.join(source_folder, patient))
+		for xray in xray_types:
+			if(xray.is_dir()):
+				for img in os.scandir(xray.path):
+					if(img.name.endswith(('tif','tiff'))):
```
Comment (zgypa) [inline bbc_clean/treerez/treerez.py:39]: What about upper case extensions? Will this catch *.TIF files as well? If not just make it case insensitive.

**Review by zgypa — CHANGES_REQUESTED** [review]

File: bbc_clean/treerez/requirements.txt (hunk 3 above)
Comment (frenchfaso) [reply bbc_clean/treerez/requirements.txt:1]: Python module dependencies can be quite finicky... I would rather be explicit about versions we know for sure work happily together.

**Review by zgypa — APPROVED** [review]

**Pull request merged**